### ISSUE TITLE
contrast for .wdn_list_descriptor

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#704]
WCAG2AA
Discovered with: http://squizlabs.github.io/HTML_CodeSniffer/

```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1:1. Recommendation: change text colour to #fff.
```

Code Snippet:

```
 <h3 class="wdn_list_descriptor wdn-text-hidden">Breadcrumbs</h3>
```

This error is only visible in mobile navigation in 4.0.   It looks like the text should be hidden, because the element also has the class .wdn-text-hidden.  So, we might just need to find a better way to hide the text.  Not sure how to approach this.
